test/e2e/corspolicy/cors_policy_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build e2e
+// +build e2e
+
+/*
+Copyright 2024 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```

Comment:
I think the Knative part is missing in the license.

Reply:
What part is missing? I copied from https://github.com/knative/serving/blob/main/test/test_images/revisionfailure/revisionfailure.go

Reply:
Hm, seems that I have dreamed that part 🤷